packages/pluggable-widgets-tools/configs/rollup.config.native.js

```diff
@@ -62,7 +62,7 @@ const nativeExternal = [
     /^react-native-fast-image($|\/)/,
     /^react-native-svg($|\/)/,
     /^react-native-vector-icons($|\/)/,
-    /^react-navigation($|\/)/,
+    /^@?react-navigation($|\/)/,
```

Comment:
Given that `react-navigation` is deprecated and its latest version `5.0.0` is also available under its successor `@react-navigation/native`, can we enforce the use of the new package instead of allowing the older package to be used?

Reply:
In this case, versions prior to the react-navigation upgrade introduced in 10.6 will not be supported.